cranelift/codegen/src/isa/zkasm/lower/isle.rs

```diff
@@ -374,6 +388,12 @@ impl generated_code::Context for ZkAsmIsleContext<'_, '_, MInst, ZkAsmBackend> {
         tmp.to_reg()
     }
 
+    fn load_global_base(&mut self) -> Reg {
```

Comment:
This probably needs to be replaced with a marker rule that doesn't have a direct lowering but is instead matched in the AMode generating ISLE rule